py/ctl/tasks.py

```diff
@@ -18,7 +18,6 @@ def task_view(id):
     task_data = dao.tasks.load_one(id_clean)
     if task_data is None:
         flask.abort(404)
-        return
```

Comment:
Is this intended removal or some occasional change? :)

Reply:
individual commit for this) yes, it's unreachable return.

Reply:
A-a-a, thanks! This is my lack of experience with flask. I failed to check documentation and see that `abort` raises exception... Thanks a lot!

Reply:
Due to that return in my tests everything was okay haha. will try to correct

Reply:
Ah, thanks, I thought I can do it myself in the morning when I'm less sleepy :)